packages/expo-updates/android/build.gradle

```
@@ -1,5 +1,16 @@
+buildscript {
+  repositories {
+    mavenCentral()
+    google()
+  }
+
+  dependencies {
+    classpath "com.google.devtools.ksp:symbol-processing-gradle-plugin:1.9.23-1.0.20"
```

```suggestion
    classpath "com.google.devtools.ksp:symbol-processing-gradle-plugin:${project.ext.kspVersion}"
```

we should be better to use the centralized kspVersion from expo-modules-core: https://github.com/expo/expo/blob/359f1da6e77431f18adbaccc10b5110d13266e2d/packages/expo-modules-core/android/ExpoModulesCorePlugin.gradle#L20-L36

this should be available right after the `applyKotlinExpoModulesCorePlugin()` so we should probably move this whole `buildscript {}  block` and `apply plugin: 'com.google.devtools.ksp'` down and after https://github.com/expo/expo/blob/359f1da6e77431f18adbaccc10b5110d13266e2d/packages/expo-updates/android/build.gradle#L12

---

Had to load it within the buildscript like we do in `packages/expo-modules-core/android-annotation-processor/build.gradle` but it worked.